src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using DurableTask.Core;
+using DurableTask.Core.Settings;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation
+{
+    /// <summary>
+    /// TelemetryActivator activates Distributed Tracing. This class only works for netstandard2.0.
+    /// </summary>
+    public class TelemetryActivator : ITelemetryActivator
+    {
+        private TelemetryClient telemetryClient;
+        private IOptions<DurableTaskOptions> options;
+
+        /// <summary>
+        /// Constructor for activating Distributed Tracing.
+        /// </summary>
+        /// <param name="options">DurableTask options.</param>
+        public TelemetryActivator(IOptions<DurableTaskOptions> options)
+        {
+            this.options = options;
+        }
+
+        /// <summary>
+        /// OnSend is an action that enable to hook of sending telemetry.
+        /// You can use this property for testing.
+        /// </summary>
+        public Action<ITelemetry> OnSend { get; set; } = null;
+
+        /// <summary>
+        /// Initialize is initialize the telemetry client.
+        /// </summary>
+        public void Initialize()
+        {
+            this.SetUpDistributedTracing();
+
+            this.SetUpTelemetryClient();
+            this.SetUpTelemetryCallbacks();
+        }
+
+        private void SetUpDistributedTracing()
```

Comment:
I feel like this whole class is setting up distributed tracing, so I think we should find a more specific name here. Maybe `LoadConfig` would be better?

Reply:
Moving to ToDos issue